transformation/src/main/java/org/opensearch/migrations/VersionMatchers.java

```diff
@@ -0,0 +1,30 @@
+package org.opensearch.migrations;
+
+import java.util.function.Predicate;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class VersionMatchers {
+    public static final Predicate<Version> isES_6_8 = VersionMatchers.matchesMinorVersion(Version.fromString("ES 6.8"));
+    public static final Predicate<Version> isES_7_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 7.10"));
```

Comment:
Actually, we need to be more specific here.  Index templates were introduced in ES 7.8, for example, so loading a 7.6 snapshot could have... undesired... results.  I'd recommend a matcher for 7.10+.

Reply:
Added `equalOrGreaterThanES_7_10`